lib/util.js

```diff
@@ -282,27 +282,30 @@ const util = {
 
       // Mapping for copying Brave's Android resource into chromium folder.
       const copyAndroidResourceMapping = {
-        [androidIconSource]: androidIconDest,
-        [androidIconSource]: androidIconDestAdditional,
-        [androidResSource]: androidResDest,
-        [androidResNightSource]: androidResNightDest,
-        [androidNtpTilesResSource]: androidNtpTilesResDest,
-        [androidResTemplateSource]: androidResTemplateDest
+        [androidIconSource]: [androidIconDest, androidIconDestAdditional],
+        [androidResSource]: [androidResDest],
+        [androidResNightSource]: [androidResNightDest],
+        [androidNtpTilesResSource]: [androidNtpTilesResDest],
+        [androidResTemplateSource]: [androidResTemplateDest]
       }
 
       console.log('copy Android app icons and app resources')
-      Object.entries(copyAndroidResourceMapping).map(([sourcePath, destPath]) => {
+      Object.entries(copyAndroidResourceMapping).map(([sourcePath, destPaths]) => {
         let androidSourceFiles = []
         if (fs.statSync(sourcePath).isDirectory()) {
           androidSourceFiles = util.walkSync(sourcePath)
         } else {
           androidSourceFiles = [sourcePath]
         }
 
-        for (const androidSourceFile of androidSourceFiles) {
-          let destinationFile = path.join(destPath, path.relative(sourcePath, androidSourceFile))
-          if (!fs.existsSync(destinationFile) || util.calculateFileChecksum(androidSourceFile) != util.calculateFileChecksum(destinationFile)) {
-            fs.copySync(androidSourceFile, destinationFile)
+        for (const destPath of destPaths) {
+          for (const androidSourceFile of androidSourceFiles) {
+            let destinationFile = path.join(destPath, path.relative(sourcePath, androidSourceFile))
+            console.log('!!!androidSourceFiles == ' + androidSourceFile)
```

Comment:
Do we still want the log messages in there?

Reply:
ugh, removing, nice catch